lang/user/id.xml

```diff
@@ -53,46 +49,46 @@
 	</about>
 
 	<app_context>
-		<check_app_state>Check Aplikasi state</check_app_state>
-		<copy_vita3k_summary>Copy Ringkasan Vita3K</copy_vita3k_summary>
-		<open_state_report>Buka state laporan</open_state_report>
+		<check_app_state>Cek state Aplikasi</check_app_state>
+		<copy_vita3k_summary>Salin Ringkasan Vita3K</copy_vita3k_summary>
+		<open_state_report>Buka laporan state</open_state_report>
 		<create_state_report>Bikin state laporan</create_state_report>
-		<copy_app_info>Copy Aplikasi Info</copy_app_info>
-		<name_and_id>Nama Dan Judul ID</name_and_id>
-		<app_summary>Ringkasan Aplikasi</app_summary>
+		<update_database>Perbarui database</update_database>
+		<copy_app_info>Salin info Apl</copy_app_info>
+		<name_and_id>Nama dan Judul ID</name_and_id>
+		<app_summary>Ringkasan Apl</app_summary>
 		<custom_config>Kastom Config</custom_config>
 		<create>Bikin</create>
 		<remove>Hapus</remove>
 		<open_folder>Buka Folder</open_folder>
 		<license>lisensi</license>
-		<shader_cache>Cache Shader</shader_cache>
-		<shader_log>Catatan Shader</shader_log>
+		<shaders_cache>Cache Shader</shaders_cache>
+		<shaders_log>Catatan Shader</shaders_log>
 		<update_history>History Update</update_history>
 		<history_version>Versi {}</history_version>
 		<information>Informasi</information>
 		<delete>
 			<app_delete>Aplikasi ini dan semua data terkait, termasuk data yang disimpan, akan dihapus.</app_delete>
-			<app_delete_note>Menghapus aplikasi mungkin memakan waktu cukup lama
-tergantung pada ukuran dan perangkat keras Kamu.</app_delete_note>
+			<app_delete_description>Menghapus aplikasi mungkin memakan waktu cukup lama
+tergantung pada ukuran dan perangkat keras Kamu.</app_delete_description>
 			<addcont_delete>Apakah Kamu ingin menghapus data yg dimasukkan ini?</addcont_delete>
 			<license_delete>Apakah kamu ingin menghapus lisensi ini?</license_delete>
-			<save_delete>Apakah Kamu ingin menghapus data tersimpan ini?</save_delete>
+			<saved_data_delete>Apakah Kamu ingin menghapus data tersimpan ini?</saved_data_delete>
 		</delete>
 		<info>
-			<eligible>Memenuhi syarat</eligible>
-			<ineligible>Tidak memenuhi syarat</ineligible>
+			<eligible>Memenuhi Syarat</eligible>
+			<ineligible>Tidak Memenuhi Syarat</ineligible>
 			<level>Level</level>
 			<name>Nama</name>
 			<trophy_earning>Perolehan Trofi</trophy_earning>
-			<parental_controls>Pengawasan ortu</parental_controls>
-			<updated>Updated</updated>
+			<parental_controls>Pengawasan Ortu</parental_controls>
+			<updated>Telah Update</updated>
 			<size>Ukuran</size>
 			<version>Versi</version>
 			<title_id>Judul ID</title_id>
 			<last_time_used>Waktu Terakhir Perguna</last_time_used>
-			<never>Gak Ada</never>
 			<time_used>Waktu Perguna</time_used>
-			<never>Ga ada</never>
+			<never>Ga Ada</never>
```

Comment:
"Tidak Ada". Actually, idk what's the context used on this string. If the context is "Never forget this configuration", the "Never" here translates to "Jangan".

Reply:
Gak ada Just make it better nowadays

Reply:
> "Tidak Ada". Actually, idk what's the context used on this string. If the context is "Never forget this configuration", the "Never" here translates to "Jangan".

"Never" the mean is never used this application.

![image](https://github.com/Vita3K/Vita3K/assets/61804715/3e6f79fe-c529-42cf-97e4-bfeb2b98245f)